docs/examples/howto/run_ckf_auto_tuning.rst

```diff
@@ -0,0 +1,98 @@
+   $ make
+   $ source build/python/setup.sh
+
+Once this setup is ready, at each new login, just do:
+
+.. code-block:: console
+
+   $ source /cvmfs/sft.cern.ch/lcg/views/LCG_100/x86_64-centos7-gcc10-opt/setup.sh
+   $ source PYTHON_VIRTUAL_ENV/bin/activate
+   $ export PYTHONPATH= 
+   $ source build/python/setup.sh
+
+How auto-tuning works
+---------------------
+A list of parameters and their range are provided to optimization framework. The framework predict a value for each parameter and provide them to the tracking algorithm. The tracking algorithm runs with these values and return the output to the optimization framework. The optimization framework analyzes the output and provide a new set of values to tracking algorithm and this cycle continues for a number of trials. At each trial, the optimization framework computes a score based on the tracking algorithm output and tries to minimize/maximize this score by suggesting better parameter configurations. The current frameworks compute the score based on the CKF track efficiency, fake rate, duplicate rate and run-time:
+
+Score = Efficiency - (fakeRate + DuplicateRate/k_dup + run-time/k_time)
+
+where k_dup and k_time are the weights for duplicate rate and run-time. These weights play a significant role in determining the configuration of best performing parameters from the optimization frameworks. 
+ 
+The list of track seeding parameters that are auto-tuned using these frameworks are as follows:
+* maxPtScattering: upper p_T limit for scattering angle calculations
+* impactMax: maximum value of impact parameter
+* deltaRMin: minimum distance in r between two measurements within one seed
+* deltaRMax: maximum distance in r between two measurements within one seed
+* sigmaScattering: number of sigma used for scattering angle calculations
+* radLengthPerSeed: average radiation lengths of material on the length of a seed
+* maxSeedsPerSpM: number of space-points in top and bottom layers considered for compatibility with middle space-point
+* cotThetaMax: maximum cotTheta between two space-points in a seed to be considered compatible
+
+The python scripts utilizing these optimization using Optuna and Orion frameworks are present in acts/Examples/Scrips/Optimization directory.
+
+Run auto-tuning using Optuna
+----------------------------
+The Optuna auto-tuning script can be run directly by invoking:
+`` python Optuna_tuning.py``
+
+This creates a new optuna study for a given number of trials defined within the script. The direction is set to maximize which means that the framework will try to maximize the score. 
+
+.. code-block:: console
+   $ study = optuna.create_study(study_name=study_name,
+		storage="sqlite:///{}.db".format(study_name),
+		direction='maximize',
+		load_if_exists=True)
```

Comment:
You are missing a line skip between `.. code-block:: console` and the following line